frontend/app/pages/dashboard/dashboard.tsx

```diff
@@ -34,15 +73,56 @@ export function DashboardPage() {
     mutate,
   } = useQuery("/api/v1/search-profiles", undefined, suppressSWRReloading);
 
+  const {
+    data: me,
+    error: meError,
+    isLoading: meLoading,
+  } = useQuery("/api/v1/users/me");
```

Comment:
```suggestion
const { loading, user: me } = useAuthorization();
```

this uses the already fetched user -> no need to refetch here -> lets safe some api calls